apps/wallet-dashboard/components/Dialogs/Assets/views/DetailsView.tsx

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import { ExplorerLinkType, useNftDetails } from '@iota/core';
+import {
+    Button,
+    ButtonType,
+    Header,
+    KeyValueInfo,
+    VisualAssetCard,
+    VisualAssetType,
+} from '@iota/apps-ui-kit';
+import Link from 'next/link';
+import { formatAddress } from '@iota/iota-sdk/utils';
+import { Layout, LayoutBody, LayoutFooter } from '../../Staking/views/Layout';
+import { IotaObjectData } from '@iota/iota-sdk/client';
+import { Collapsible } from '@/components/Collapsible/Collapsible';
+import { ExplorerLink } from '@/components/ExplorerLink/ExplorerLink';
+import { useCurrentAccount } from '@iota/dapp-kit';
+
+interface DetailsViewProps {
+    asset: IotaObjectData;
+    handleClose: () => void;
+}
+
+export function DetailsView({ handleClose, asset }: DetailsViewProps) {
+    const account = useCurrentAccount();
+
+    const senderAddress = account?.address ?? '';
+    const objectId = asset.objectId;
+
+    const {
+        nftName,
+        nftImageUrl,
+        nftMeta,
+        ownerAddress,
+        isTransferable,
+        metaKeys,
+        metaValues,
+        formatMetaValue,
+
+        isContainedInKiosk,
+        kioskItem,
+    } = useNftDetails(objectId, senderAddress);
+
+    function handleMoreAboutKiosk() {
+        window.open('https://wiki.iota.org/', '_blank');
+    }
+
+    function handleMarketplace() {
+        window.open('https://wiki.iota.org/', '_blank');
+    }
+
+    function handleSend() {
+        console.log('send');
```

Comment:
I think you have to add here the logic to send :trollface:

Reply:
issue is just styling, maybe we can do that in another PR

Reply:
but the logic was before